### PR TITLE
Adding a group description

### PR DESCRIPTION
**Pull request by rosasck**
Cant be pushed until storage service is deployed for this to work, will fail tests until then

### CODE REVIEW

File: lib/storage/client.js [hunk 1]
```diff
@@ -1055,10 +1055,11 @@ class Client extends CryptoConsumer {
    *
    * @param {string} name  the name of the group
    * @param {array} capabilities an array of strings of the capabilities the group will have
+   * @param {string} description  details about the group
    *
    * @returns {Group} A response from TozStore; the group that has been written.
    */
-  async createGroup(name, capabilities = []) {
+  async createGroup(name, capabilities = [], description) {
```
Comment (efabens) [inline lib/storage/client.js:1062]: We might want description to have a sane default. either an empty string or "A TozStore group"

**Comment by rosasck** [comment]
added null as a place holder for description and changed json tag from group_description to description

**Review by lkwdwrd — APPROVED** [review]
Code looks good, but I see for some reason tests are failing when run in the Windows environment. When groups is fully ready and merged down, before we cut the new SDK release lets make sure the tests are passing first.

**Review by willmichael — APPROVED** [review]
Change looks simple enough, but do we need to test that group descriptions are returned when we access groups? (ignore me if i missed that in the tests)

**Pull request merged**